explainer.md

```diff
@@ -483,6 +483,31 @@ vrSession.addEventListener('resetpose', vrSessionEvent => {
 });
 ```
 
+### Handling Transparent Displays
```

Comment:
We discussed needing to know _how_ to specify a pixel should be transparent. Unless we want to commit to a specific color or alpha value - something that seems bound to be wrong on some hardware - this attribute should tell the app how to render transparent. (That may also allow non-transparent displays to pretend to be transparent by, for example, passing through video to such pixels. Such functionality would probably need to be requested somehow, though.)

Also, if some color or alpha value is going to render as transparent (and this is not clearly obvious), we need a way to indicate to authors that they should not use such values. This seems problematic for apps that are just trying to write a generic WebVR app. (This may also warrant some type of feature detection and/or request to "prevent me from drawing transparent pixels.)